ui/src/Components/CreateProfile.jsx

```diff
@@ -142,39 +142,39 @@ class CreateProfile extends React.Component {
     render() {
         return (
             <>
-            <NavBar/>
-            <Form onSubmit={ this.handleSubmit }>
+            <AvForm onSubmit={ this.handleSubmit }>
                 <FormGroup row>
                 <Label for="name" sm={2}>Complete Profile</Label>
                 </FormGroup>
                 <FormGroup row>
                     <Label for="name" sm={2}>First Name</Label>
                     <Col sm={10}>
-                    <Input type="text" name="name" id="firstname" placeholder="John" onChange={this.handleFirstNameChange} value={this.state.firstName}/>
+                    <AvField name="name" type="text" placeholder="John" onChange={this.handleFirstNameChange} value={this.state.firstName} required />
+                    {/* <Input type="text" name="name" id="firstname" placeholder="John" onChange={this.handleFirstNameChange} value={this.state.firstName}/> */}
                     </Col>
                 </FormGroup>
                 <FormGroup row>
                     <Label for="name" sm={2}>Last Name</Label>
                     <Col sm={10}>
-                    <Input type="text" name="name" id="lastname" placeholder="Doe" onChange={this.handleLastNameChange} value={this.state.lastName}/>
+                    <AvField name="name" type="text" placeholder="Doe" onChange={this.handleLastNameChange} value={this.state.lastName} required />
                     </Col>
                 </FormGroup>
                 <FormGroup row>
                     <Label for="email" sm={2}>Email</Label>
                     <Col sm={10}>
-                    <Input type="email" name="email" id="email" placeholder="johndoe@gmail.com" onChange={this.handleEmailChange} value={this.state.email}/>
+                    <AvField name="email" type="email" placeholder="johndoe@gmail.com" onChange={this.handleEmailChange} value={this.state.email} required />
                     </Col>
                 </FormGroup>
                 <FormGroup row>
                     <Label for="phone" sm={2}>Phone Number</Label>
                     <Col sm={10}>
-                    <Input type="tel" name="phone" id="phone" placeholder="415-879-7877" onChange={this.handleNumChange} value={this.state.number}/>
+                    <AvField name="phone" type="tel" placeholder="415-879-7877" onChange={this.handleNumChange} value={this.state.number} required />
```

Comment:
why are we wrapping these with an AvField? Is this part of the auth?

Reply:
This is form validation, feel free to test it out! :)